tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -136,8 +138,12 @@ protected:
    /// \return `entryNumber` if the entry was successfully loaded, `kInvalidNTupleIndex` otherwise.
    virtual NTupleSize_t LoadEntry(NTupleSize_t entryNumber) = 0;
 
-   RNTupleProcessor(const std::vector<RNTupleOpenSpec> &ntuples)
-      : fNTuples(ntuples), fNEntriesProcessed(0), fCurrentNTupleNumber(0), fCurrentEntryNumber(0)
+   RNTupleProcessor(const std::vector<RNTupleOpenSpec> &ntuples, std::unique_ptr<RNTupleModel> model)
+      : fNTuples(ntuples),
+        fModel(std::move(model)),
+        fNEntriesProcessed(0),
```

Comment:
These default initializations should be inlined with the field declarations

EDIT: I see this is done in the following commit; consider squashing the changes